src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyPushOnlyView.js

```diff
@@ -0,0 +1,53 @@
+import { createButton, loc } from 'okta';
+import { BaseForm } from '../../internals';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+
+const Body = BaseForm.extend(Object.assign({
+  noButtonBar: true,
+
+  initialize() {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    this.addView();
+  },
+
+  title() {
+    return loc('oie.okta_verify.push.title', 'login');
+  },
+
+  addView() {
+    this.add(createButton({
+      className: 'button button-wide button-primary send-push',
+      title: loc('oie.okta_verify.sendPushButton', 'login'),
+      click: () => {
+        this.$el.submit();
+      },
+    }));
+    this.add(
+      `<span class='accessibility-text' role='alert'>${loc('oie.okta_verify.sendPushButton', 'login')}</span>`,
+    );
+  },
+
+  render() {
+    BaseForm.prototype.render.apply(this, arguments);
+    this.$el.addClass('okta-verify-push-challenge');
+    // Move checkbox below the button
+    // Checkbox is rendered by BaseForm using remediation response and 
+    // hence by default always gets added above buttons.
+    const checkbox = this.$el.find('[data-se="o-form-fieldset-authenticator.autoChallenge"]');
+    checkbox.length && this.$el.find('.o-form-fieldset-container').append(checkbox);
+  },
+
+  getUISchema() {
+    const uiSchemas = BaseForm.prototype.getUISchema.apply(this, arguments);
+    return uiSchemas.filter(schema => schema.name !== 'authenticator.methodType');
+  },
+}));
+
+const AuthenticatorView = BaseAuthenticatorView.extend({
```

Comment:
similar to my other comment, seems that we don't really need to create a View here, so exporting the Body instead should do the trick
```
export default Body;
```
and we could even rename this to "ChallengeAuthenticatorDataOktaVerifyPushOnlyBody.js"